pkg/controller/component/component_resources.go

```diff
@@ -121,11 +126,13 @@ func (r *ReconcileComponent) newBuildConfig(cr *devconsoleapi.Component, builder
 
 func newDeploymentConfig(cr *devconsoleapi.Component, output *imagev1.ImageStream) *v1.DeploymentConfig {
```

Comment:
`Annotations` can also be added to `ObjectMeta`:

```
ObjectMeta: metav1.ObjectMeta{                                                                                                                            
                           Name:      cr.Name,                                                                                                                               
                           Namespace: cr.Namespace,                                                                                                                          
                           Labels:    labels,      
                           Annotations: annotations,                                                                                                      
             },   
```



at line 147 (https://github.com/redhat-developer/devconsole-operator/pull/122/files#diff-3240f219b66b3cb310b8835fe5cf7e42R144)

Reply:
Good catch @DhritiShikhar will update!

Reply:
fixed here: e3c82fdee4dab0d861e8ee8e974bea9795a64c4d